File: event/src/test/java/de/danoeh/antennapod/event/playback/PlaybackHistoryEventTest.java
package de.danoeh.antennapod.event.playback;

import org.junit.Test;

import static org.junit.Assert.assertEquals;
import static org.junit.Assert.assertNotSame;

public class PlaybackHistoryEventTest {

    @Test
    public void listUpdatedReturnsNewInstanceEachCall() {
        PlaybackHistoryEvent first = PlaybackHistoryEvent.listUpdated();
        PlaybackHistoryEvent second = PlaybackHistoryEvent.listUpdated();
        assertNotSame(first, second);
    }

    @Test
    public void toStringIsFixedString() {
        assertEquals("PlaybackHistoryEvent", PlaybackHistoryEvent.listUpdated().toString());
    }
}

File: event/src/test/java/de/danoeh/antennapod/event/playback/PlaybackHistoryEventTest.kt
package de.danoeh.antennapod.event.playback

import org.junit.Assert.assertEquals
import org.junit.Assert.assertNotSame
import org.junit.Test

class PlaybackHistoryEventTest {

    @Test
    fun listUpdatedReturnsNewInstanceEachCall() {
        val first = PlaybackHistoryEvent.listUpdated()
        val second = PlaybackHistoryEvent.listUpdated()
        assertNotSame(first, second)
    }

    @Test
    fun toStringIsFixedString() {
        assertEquals("PlaybackHistoryEvent", PlaybackHistoryEvent.listUpdated().toString())
    }
}
